modules/sdk-coin-polygon/src/polygon.ts

```diff
@@ -325,7 +325,7 @@ export class Polygon extends Eth {
    * @param {String} params.recoveryDestination target address to send recovered funds to
    * @returns {Promise<RecoveryInfo | OfflineVaultTxInfo>}
    */
-  async recover(params: RecoverOptions): Promise<RecoveryInfo | OfflineVaultTxInfo> {
+  async recoverEth(params: RecoverOptions): Promise<RecoveryInfo | OfflineVaultTxInfo> {
```

Comment:
This is a breaking change. Do you want to do a full version bump or can this stay as the same method name?

Reply:
I think this needs to change to `recoverEth` in order to correctly support the recover flow for TSS recoveries. Right now, it should inherit the `recover` method from Eth controller, so users should still be using the `recover` method in the SDK and pass in their usual params. Ideally, it shouldn't be a breaking change, because it should still work with the exact same params and the same method name (it inherits `recover` from eth). I can make this method private for clarification though. Unless I'm missing something>

Reply:
This is a published method, so if you are going to remove it, it needs to be recorded as a breaking change.

Reply:
Okay, even if `polygon.recover` is still supported because it's inherited from Eth?

Reply:
If the inherited method is working the way you want it to work, then yes, you are good to go.

Reply:
`recover` method would be used in WRW, so this would break WRW if we bump sdk
also `recoverEth` doesn't make sense in polygon class

Reply:
`recover` method should still be inherited from the eth class so that should still work as intended (unit tests for `recover` all still pass). But the naming is fair, maybe something like `recoverEthLike` is more apt. Can change that.